core/src/main/java/com/linecorp/armeria/server/logging/AccessLogType.java

```diff
@@ -125,7 +144,7 @@ public boolean isConditionAvailable() {
         return isConditionAvailable;
     }
 
-    public boolean isVariableRequired() {
-        return isVariableRequired;
+    public VariableRequirement variableRequirement() {
```

Comment:
`VariableRequirement` looks like a package-local class. Should this method be also package-local?

Reply:
Got it.